x/profiles/keeper/relay_chain_links.go

```diff
@@ -61,13 +61,6 @@ func (k Keeper) OnRecvLinkChainAccountPacket(
 			hex.EncodeToString(profile.GetPubKey().Bytes()), hex.EncodeToString(pubKey.Bytes()))
 	}
 
-	// Verify the source proof by making sure the user has signed the destination
-	// address using the source address private key
-	err = data.SourceProof.Verify(k.cdc, k.legacyAmino, data.DestinationAddress, srcAddrData)
-	if err != nil {
-		return packetAck, err
-	}
```

Comment:
This check is duplicated with `SaveChainLink` to verify the source proof:
https://github.com/desmos-labs/desmos/blob/20e9380bd9ac0847ac1a0b3de71fd1cde598e29d/x/profiles/keeper/keeper_chain_links.go#L38-L39